tests/no-global-selector.js

```diff
@@ -16,6 +16,7 @@ ruleTester.run( 'no-global-selector', rule, {
 		'$(".selector", context)',
 		'$(".selector", "#")',
 		'$(".selector", [])',
+		'$("#id")',
```

Comment:
Per discussion on PR, this should be configurable, e.g. { "allowIds": true }, but default to false